packages/components/tests/integration/components/hds/alert/index-test.js

```diff
@@ -172,6 +173,19 @@ module('Integration | Component | hds/alert/index', function (hooks) {
     assert.dom('#test-alert').hasAttribute('aria-labelledby', 'custom-id');
   });
 
+  test('it should pass available a11y automation tests', async function (assert) {
+    await render(
+      hbs`<Hds::Alert @type="inline" id="test-alert" as |A|><A.Title id="custom-id">This is the title</A.Title><A.Button @text="I am a button" @size="small" /></Hds::Alert>`
+    );
+
+    let axeOptions = {
+      runOnly: ['wcag21a', 'wcag21aa', 'best-practice'],
+    };
```

Comment:
I wonder if we could store this at a higher level, so we consistently test against the same criteria.

Reply:
@alex-ju perhaps the `test-helper.js` file that's already in the tests folder. WDYT?

Reply:
yep, that looks like a good place!